page_rules.go

```diff
@@ -109,7 +109,7 @@ type PageRule struct {
 	Targets    []PageRuleTarget `json:"targets"`
 	Actions    []PageRuleAction `json:"actions"`
 	Priority   int              `json:"priority"`
-	Status     string           `json:"status"` // can be: active, paused
+	Status     string           `json:"status"` // can be: active, disabled
```

Comment:
I wonder if we're better off removing this code comment and pushing people to the API docs instead 🤔